pkgs/tools/misc/goss/default.nix

```
@@ -26,17 +32,30 @@ buildGoModule rec {
     "-s" "-w" "-X main.version=v${version}"
   ];
 
+  nativeBuildInputs = [ makeWrapper ];
+
   checkFlags = [
     # Prometheus tests are skipped upstream
     # See https://github.com/goss-org/goss/blob/master/ci/go-test.sh
     "-skip" "^TestPrometheus"
   ];
 
+  postInstall = let
+    runtimeDependencies = [ bash getent ]
+      ++ lib.optionals (!stdenv.isDarwin) [ systemd ];
```

Will be helpful once the BSDs get fully up and running

```suggestion
      ++ lib.optionals stdenv.isLinux [ systemd ];
```